app/governance/components/votingChart/voteGraph.tsx

```diff
@@ -0,0 +1,405 @@
+              className={styles.graph}
+              style={{
+                height: abstainHeight,
+              }}
+            ></div>
+          )}
+          <div className={styles.amountInfo}>
+            <div>
+              <Text size="x-sm">
+                {formatBalance(abstainVotes.toString(), 0, { short: true })}{" "}
+              </Text>
+            </div>
+            <div className={styles.icon}>
+              <Icon
+                icon={{
+                  url: "/tokens/canto.svg",
+                  size: {
+                    width: 12,
+                    height: 12,
+                  },
+                }}
+                themed
+              />
+            </div>
+          </div>
+          {abstainHeight == 0 && (
+            <div
+              style={{
+                height: 200,
+              }}
+            />
+          )}
+        </div>
+      </div>
+      <div className={styles.inforow}>
+        <div className={styles.voteOption}>
+          <div className={styles.circleContainer}>
+            <div
+              className={styles.circle}
+              style={{ backgroundColor: "green" }}
+            />
+          </div>
+          <div className={styles.option}>
+            <Text font="proto_mono" size="xx-sm">
+              YES ({yesPercentage.toFixed(1)}%)
+            </Text>
+          </div>
+        </div>
+        <div className={styles.voteOption}>
+          <div className={styles.circleContainer}>
+            <div className={styles.circle} style={{ backgroundColor: "red" }} />
+          </div>
+          <div className={styles.option}>
+            <Text font="proto_mono" size="xx-sm">
+              NO ({noPercentage.toFixed(1)}%)
+            </Text>
+          </div>
+        </div>
+        <div className={styles.voteOption}>
+          <div className={styles.circleContainer}>
+            <div
+              className={styles.circle}
+              style={{ backgroundColor: "purple" }}
+            />
+          </div>
+          <div className={styles.option}>
+            <div>
+              <Text font="proto_mono" size="xx-sm">
+                VETO ({vetoPercentage.toFixed(1)}%)
+              </Text>
+            </div>
+          </div>
+        </div>
+        <div className={styles.voteOption}>
+          <div className={styles.circleContainer}>
+            <div
+              className={styles.circle}
+              style={{ backgroundColor: "yellow" }}
+            />
+          </div>
+          <div className={styles.option}>
+            <div>
+              <Text font="proto_mono" size="xx-sm">
+                ABSTAIN ({abstainPercentage.toFixed(1)}%)
+              </Text>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export const VoteBar = ({
+  height,
+  maxHeight,
+  index,
+}: {
+  height: number;
+  maxHeight: number;
+  index: number;
+}) => {
+  const getColor = (index: number): string => {
```

Comment:
again not sure why we are passing inbuilt colors. we already have custom colors

Reply:
Yeah changed these colors to the ones in design